src/pages/devtools/componnents/FormsValuesReader.tsx

```diff
@@ -0,0 +1,58 @@
+import { useState, useEffect } from 'react';
```

Comment:
This component should not be placed in the `src/pages/devtools/` It is used only in the `Panel` so it should be placed in the `src/pages/panel/componets` instead